src/modules/package.rs

```diff
@@ -57,6 +65,11 @@ fn get_package_version() -> Option<String> {
         return extract_package_version(&package_json);
     }
 
+    let poetry_toml = utils::read_file("pyproject.toml");
+    if let Ok(poetry_toml) = poetry_toml {
+        return extract_poetry_version(&poetry_toml);
+    }
+
```

Comment:
Would it be better if
```rust
    if let Ok(cargo_toml) = utils::read_file("Cargo.toml") {
        extract_cargo_version(&cargo_toml)
    } else if let Ok(package_json) = utils::read_file("package.json") {
        extract_package_version(&package_json)
    } else if let Ok(poetry_toml) = utils::read_file("pyproject.toml") {
        extract_poetry_version(&poetry_toml)
    } else {
        None
    }
```

Reply:
I personally prefer how it's read without the `else if`s, but it's functionally the same. It's up to @LinusU.  😄

Reply:
I committed the suggested change. It is slightly shorter now, which will be beneficial if more package types are added.